src/Skyroom/Controller/SkyroomController.php

```diff
@@ -76,12 +68,17 @@ function parseRequest($do, $wp)
                 $skyroomRoomId = $product->getSkyroomId();
 
                 try {
-                    $this->userRepository->ensureSkyroomUserAdded(wp_get_current_user());
-                    $skyroomUserId = $this->userRepository->getSkyroomId(get_current_user_id());
+                    $userData = wp_get_current_user();
+
+                    $username = $userData->data->user_login;
+                    $nickname = $userData->data->display_name;
 
-                    $url = $this->client->request('getLoginUrl', [
+                    $url = $this->client->request('createLoginUrl', [
                         'room_id' => $skyroomRoomId,
-                        'user_id' => $skyroomUserId,
+                        'user_id' => $username,
```

Comment:
Maybe better to use wordpress user id for this field, some wordpress sites don't require username on signup.

Reply:
I think if username is, the user can recognize better And we had his request

Reply:
Yeah makes sense, but what about wordpress sites that don't take username on signup? You can add id to statement as fallback value.

Reply:
We do not have such a site!
If the user is not registered, he can not buy the product